README.md

```diff
@@ -4,6 +4,13 @@ This module allows you to integrate AWS master account with Uptycs so required A
 
 After AWS Master Account is integrated with Uptycs, any AWS Child accounts under Organization are automatically integrated (More on this later).
 
+## Prerequisites
+
+- Requires Terraform version >= 1.2.0
+- The user should have Admin access on the master account
+- Every child account in the organization should have OrganizationAccountAccessRole role. Please carefully set up this role in each child account before beginning the integration process.
+- When applicable, S3 Bucket for CloudTrail should be owned by the Master account
```

Comment:
What is the purpose of this?
We should remove this.
We are saying "when applicable". What if they don't?
This totally wrong to say when we support alternatives.